src/test/java/org/mule/extension/db/internal/CreateArrayTestCase.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.internal;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mule.extension.db.api.param.JdbcType.BLOB;
+import static org.mule.extension.db.api.param.JdbcType.CLOB;
+import static org.mule.extension.db.internal.domain.connection.oracle.OracleConnectionUtils.getOwnerFrom;
+import static org.mule.extension.db.internal.domain.connection.oracle.OracleConnectionUtils.getTypeSimpleName;
+import static org.mule.extension.db.internal.domain.connection.oracle.OracleDbConnection.ATTR_NO_PARAM;
+import static org.mule.extension.db.internal.domain.connection.oracle.OracleDbConnection.ATTR_TYPE_NAME_PARAM;
+import static org.mule.extension.db.internal.domain.connection.oracle.OracleDbConnection.QUERY_OWNER_CONDITION;
+import static org.mule.extension.db.internal.domain.connection.oracle.OracleDbConnection.QUERY_TYPE_ATTRS;
+import static org.mule.extension.db.internal.domain.connection.type.resolver.CollectionTypeResolver.QUERY_ALL_COLL_TYPES;
+import org.mule.extension.db.internal.domain.connection.DefaultDbConnection;
+import org.mule.extension.db.internal.domain.connection.oracle.OracleDbConnection;
+
+import com.mysql.jdbc.PreparedStatement;
+
+import java.sql.Array;
+import java.sql.Blob;
+import java.sql.Clob;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+public class CreateArrayTestCase extends AbstractDbFunctionTestCase {
```

Comment:
Can you explain what is this testing? Everything is mocked, so you don't really know if this is actually works against different databases.